### PR TITLE
DEBUG-2334 Ruby DI documentation page

### PR DESCRIPTION
**Pull request by p-datadog**
### What does this PR do? What is the motivation?

Adds Ruby to the dynamic instrumentation languages.

### Merge instructions

Merge readiness:
- [ ] Ready for merge

Merge queue is enabled in this repo. To have it automatically merged after it receives the required reviews, create the PR (from a branch that follows the `<yourname>/description` naming convention) and then add the following PR comment:

```
/merge
```

### CODE REVIEW

**Comment by janine-c** [comment]
Hey @p-datadog 👋🏻 To check and merge your branch into the documentation repo, the branch name has to follow our [branch naming convention](https://datadoghq.atlassian.net/wiki/spaces/docs4docs/pages/2231206099/Docs+Build#Branch-naming-convention). Could you please close this PR and reopen it with a branch name that has your Slack username in it, followed by a slash? Sorry for the confusion!

**Comment by rtrieu** [comment]
Created DOCS-9777 for docs review.

**Comment by rtrieu** [comment]
Created DOCS-9777 for docs review.

File: content/en/dynamic_instrumentation/enabling/ruby.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+## Configuration
+
+Configure Dynamic Instrumentation using the following environment variables:
+
+| Environment variable                             | Type          | Description                                                                                                               |
+| ------------------------------------------------ | ------------- | ------------------------------------------------------------------------------------------------------------------------- |
+| `DD_DYNAMIC_INSTRUMENTATION_ENABLED`             | Boolean       | Set to `true` to enable Dynamic Instrumentation.                                                                          |
+| `DD_SERVICE`                                     | String        | The [service][4] name, for example, `web-backend`.                                                                        |
+| `DD_ENV`                                         | String        | The [environment][4] name, for example: `production`.                                                                     |
```
Comment (brett0000FF) [inline content/en/dynamic_instrumentation/enabling/ruby.md:37]:
```suggestion
| `DD_ENV`                                         | String        | The [environment][4] name, for example, `production`.                                                                     |
```

File: content/en/dynamic_instrumentation/enabling/ruby.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+| `DD_VERSION`                                     | String        | The [version][4] of your service.                                                                                         |
+| `DD_TAGS`                                        | String        | Tags to apply to produced data. Must be a list of `<key>:<value>` separated by commas such as: `layer:api,team:intake`.   |
+
+## What to do next
+
+See [Dynamic Instrumentation][5] for information about setting snapshot and metric probes and browsing and indexing the data.
+
+## Further Reading
```
Comment (brett0000FF) [inline content/en/dynamic_instrumentation/enabling/ruby.md:45]:
```suggestion
## Further reading
```

**Review by brett0000FF — CHANGES_REQUESTED** [review]
@p-datadog Thanks for adding this content! It looks good--just had a few small suggestions.

Requesting changes because we need to rename the branch in order to merge this. Could you please re-open with your branch in `<username>/<branch-name>` format? Otherwise, our build preview won't run, and this blocks our merge queue. Thanks!

**Comment by p-datadog** [comment]
Now in https://github.com/DataDog/documentation/pull/26924

**Pull request closed without merging**